grails-app/domain/imod/LearningObjective.groovy

```diff
@@ -64,9 +64,7 @@ class LearningObjective {
 	]
 
 	public static final List genericConditions = [
-		'Given a problem specification',

```

Comment:
We do want to keep Generic Conditions,
so these lines should be kept,
and some logic should be added around
https://github.com/IMOD-ASU/imod/blob/development/grails-app/views/learningObjective/condition.gsp#L33
checking if value is in the genericConditions list